src/main/resources/assets/corpse/lang/en_us.json

```diff
@@ -1,21 +1,21 @@
 {
   "entity.corpse.corpse": "Corpse",
   "entity.corpse.corpse_of": "Corpse of %s",
-  "button.corpse.previous": "Previous",
+  "button.corpse.previous": "Back",
   "button.corpse.next": "Next",
   "gui.corpse.page": "%s of %s",
-  "gui.corpse.death_history.title": "Death History",
+  "gui.corpse.death_history.title": "Player Death History",
   "gui.corpse.death_history.name": "Name",
   "gui.corpse.death_history.date_format": "yyyy/MM/dd HH:mm:ss",
   "gui.corpse.death_history.dimension": "Dimension",
   "gui.corpse.death_history.location": "Location",
   "button.corpse.show_items": "Items",
-  "key.corpse.death_history": "Open Death History",
-  "message.corpse.no_death_history": "No Death History",
+  "key.corpse.death_history": "Open Player Death History",
+  "message.corpse.no_death_history": "No player death history",
   "chat.corpse.teleport_death_location": "Teleport to death location %s",
   "tooltip.corpse.teleport": "Teleport",
   "tooltip.corpse.death_date": "Died on %s",
-  "tooltip.corpse.item_count": "Containing %s stack(s)",
+  "tooltip.corpse.item_count": "Contains %s stack(s)",
```

Comment:
Basically the same, just a matter of preference.

Reply:
Present simple is simpler than imperfect participle, so it should be more understandable to people who are forced to use the default U.S. English translation of your mod. Simpler may also mean easier to translate.